client/network/test/src/sync.rs

```diff
@@ -814,3 +811,32 @@ fn sync_to_tip_requires_that_sync_protocol_is_informed_about_best_block() {
 	// However peer 1 should still not have the block.
 	assert!(!net.peer(1).has_block(&block_hash));
 }
+
+#[test]
+fn make_sure_we_sync_to_tip_without_informing_sync_about_new_best_block() {
```

Comment:
The name of this test conflicts with the name of the test above 🙈

Reply:
Hmm 😂